### ISSUE TITLE
Zim can't started: Failed running: /home/$username/zim

### ISSUE THREAD

**Issue by danykey** [NixOS/nixpkgs#16716]
## Issue description
While starting the Zim throwed dialog ```Failed running: /home/$username/zim```.
After close the dialog a window appeared:
```
This is zim 0.63
Platform: posix
Locale: en_US UTF-8
FS encoding: UTF-8
Python: (2, 7, 11, 'final', 0)
Gtk: (2, 24, 29)
Pygtk: (2, 24, 0)
Zim revision is:
  branch: zim-trunk
  revision: 765 jaap.karssenberg@gmail.com-20150613194828-pygsvbwdn2sefqhe
  date: 2015-06-13 21:48:28 +0200

======= Traceback =======
  File "/nix/store/ydp7lkqivnl6rp0qxk6bca354ric4rch-zim-0.63/lib/python2.7/site-packages/zim/main.py", line 446, in main
    obj.run()
  File "/nix/store/ydp7lkqivnl6rp0qxk6bca354ric4rch-zim-0.63/lib/python2.7/site-packages/zim/main.py", line 211, in run
    zim.ipc.start_server_if_not_running()
  File "/nix/store/ydp7lkqivnl6rp0qxk6bca354ric4rch-zim-0.63/lib/python2.7/site-packages/zim/ipc.py", line 283, in start_server_if_not_running
    raise AssertionError, 'Failed to start server (spawning)'
AssertionError: Failed to start server (spawning)
```
### Steps to reproduce
Just install and try to run.

## Technical details

* System: 16.03.923.9e959ce (Emu)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.03.923.9e959ce"

**Comment by bjornfor** [comment]
Confirmed (on NixOS 16.03, zim 0.63). A workaround, found after reading [1]: ```(cd $(dirname $(command -v zim)) && ./zim)```

[1] https://bbs.archlinux.org/viewtopic.php?id=150810

**Comment by bjornfor** [comment]
CC @vbgl, @pSub, @jraygauthier (from git log)

**Comment by vbgl** [comment]
I think this has been addressed by 13c6b9ba8549c3db91c6f321e0413a538bce9388.

An other workaround is to run zim in stand-alone mode rather than in client/server mode (the error message reads: “failed to start server”) with `zim --standalone`.

**Comment by bjornfor** [comment]
@vbgl: Thanks! I'll try backporting to stable.

Doh, I thought the OP used 16.09 (I guess I misread 16.03.9... as 16.09 or something).

**Comment by bjornfor** [comment]
Fixed: 17a777c8edefef78e50a5a976c71053ad63ad392.
